_content/gateways/faq/index.md

```diff
@@ -0,0 +1,75 @@
+
+#### Q. How do I get started with The Things Gateway?
+
+Getting started with The Things Gateway is a 3 step process:
+
+- First step - Connect the antenna and plug the gateway into a power outlet
+- Second step - Go to the website (https://ttn.fyi/activate)
+- Third step - Follow the instructions on the website
+
+
+#### Q. How can I reset my gateway?
+
+- Pressing the button shortly while powered on restarts the gateway.
+- Holding the button for 2 seconds while powered on it erased the WiFi settings.
+- Holding the button for 5 seconds while powered on it erases the WiFi settings and activation settings.
+- Holding the button while powering on it will erase the full serial flash
+
+#### Q. The second time I power my gateway, it takes time to start up. why?
+
+Upon every power cycle and every 24 hours, the gateway checks for new firmware. If the latest firmware is not downloaded yet it will download the new image and perform a reboot in order to install the image. This will take a few minutes. The process is indicated by a flashing power LED. Power cycling during this process is not dangerous, but this process will be restarted on the next start-up. So, please be patient
+
+#### Q. My gateway stays in AP mode. What do I do?
+
+This behaviour occurs when there is a glitch in the wifi connection of the gateway. When this happens, your gateway will turn into an AP(Access Point) and devices you used to set up the gateway can connect to it (your laptop or phone). To avoid this issue, you should ‘forget’ the wifi AP of the gateway on your phone/laptop.
+
+#### Q. I want to get in-depth insight/read debug messages of my gateway. Is that possible?
+The gateway has a UART port on which it prints debug messages. To see these messages you need to connect a device between the gateway and your computer that can read out a UART (e.g. an FTDI cable/board).
+
+- Connect a UART-interface device to your computer and the RX line -> RX pin and GND line -> GND pin (Please refer to the images below).
+
+    **FTDI Board**
+    ![FTDI Board](ftdi.jpg)
+
+    **UART pins on the gateway**
+    ![gtw-uart](gtw-uart.jpg)
+
+    **Sample Gateway to UART connection**
+    ![FTDI Board](sampleconn.png)
+
+
+- Start a terminal session (putty: windows, terminal: macOS/linux) to UART with 115200@8N
+
+#### Q. What do the LED lights on the gateway indicate?
+
+The LED lights are a great way of understanding the current state of the gateway. Click [here](ledstatus.md) for the complete list.
+
+
+#### Q. What if I take the gateway to a different location. Do I have repeat the configuration?
+
+Once you configure the gateway, the settings are locked to that location. So, in case you move to a different location you will have to do the activation again. 
```

Comment:
Can't you change a gateway's location in the gateway's console page, in Settings > Location?

Reply:
absolutely right..